src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "My Journey to GSoC @Layer5"
+subtitle: "A Rollercoaster experience"
+date: 2022-06-12 08:00:00 -0630
+author: Yash Kamboj
+thumbnail: ./Yash_Gsoc.png
+category: Community
+description: "Yash's journey to an GSoC @Layer5"
+tags:
+ - Community
+ - Projects
+ - News
```

Comment:
This isn't news, nor is it really about projects. Is there an internships, programs, or careers tag already?